src/components/Calculator.js

```diff
@@ -1,43 +1,66 @@
         <div className="calcKeysContainer">
           <div className="calcKeys">
             <div className="calcNumbers">
-              <button type="button">AC</button>
-              <button type="button">+/-</button>
-              <button type="button">%</button>
+              <button type="button" onClick={() => this.handleClick('AC')}>AC</button>
+              <button type="button" onClick={() => this.handleClick('+/-')}>+/-</button>
+              <button type="button" onClick={() => this.handleClick('%')}>%</button>
             </div>
             <div className="calcNumbers">
-              <button type="button">7</button>
-              <button type="button">8</button>
-              <button type="button">9</button>
+              <button type="button" onClick={() => this.handleClick('7')}>7</button>
+              <button type="button" onClick={() => this.handleClick('8')}>8</button>
+              <button type="button" onClick={() => this.handleClick('9')}>9</button>
             </div>
             <div className="calcNumbers">
-              <button type="button">4</button>
-              <button type="button">5</button>
-              <button type="button">6</button>
+              <button type="button" onClick={() => this.handleClick('4')}>4</button>
+              <button type="button" onClick={() => this.handleClick('5')}>5</button>
+              <button type="button" onClick={() => this.handleClick('6')}>6</button>
             </div>
             <div className="calcNumbers">
-              <button type="button">1</button>
-              <button type="button">2</button>
+              <button type="button" onClick={() => this.handleClick('1')}>1</button>
+              <button type="button" onClick={() => this.handleClick('2')}>2</button>
               <button type="button">3</button>
             </div>
             <div className="calcNumbers footer">
-              <button type="button">0</button>
-              <button type="button">.</button>
+              <button type="button" onClick={() => this.handleClick('0')}>0</button>
+              <button type="button" onClick={() => this.handleClick('.')}>.</button>
             </div>
           </div>
           <div className="calcSymbols">
-            <button type="button" className="calcSymbolsColor">÷</button>
-            <button type="button" className="calcSymbolsColor">×</button>
-            <button type="button" className="calcSymbolsColor">-</button>
-            <button type="button" className="calcSymbolsColor">+</button>
-            <button type="button" className="calcSymbolsColor">=</button>
+            <button type="button" className="calcSymbolsColor" onClick={() => this.handleClick('÷')}>÷</button>
+            <button type="button" className="calcSymbolsColor" onClick={() => this.handleClick('×')}>×</button>
+            <button type="button" className="calcSymbolsColor" onClick={() => this.handleClick('-')}>-</button>
```

Comment:
- [ ] When I try to multiply two numbers, I get the error that `'×'` is not an operator. Based on the logic in the **_operate.js_** file the symbol for multiplication used is `'x'` and not `'×'`. Kindly make this change by passing the right string as a parameter for the `this.handleClick` function for multiplication. 👍